### PR TITLE
MethodArgumentSpaceFixer - PHP7.3 trailing commas in function calls

### PR DESCRIPTION
**Pull request by SpacePossum**
closes https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4414

### CODE REVIEW

File: tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php [hunk 1]
```diff
@@ -192,12 +192,26 @@ public function provideFixCases()
                 ['keep_multiple_spaces_after_comma' => true],
             ],
             'skip array' => [
-                '<?php array(10 , 20 ,30);',
+                '<?php array(10 , 20 ,30); $foo = [ 10,50 , 60 ] ?>',
```
Comment (keradus) [inline tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php]: :+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Comment by brendt** [comment]
Has this been tagged?

**Comment by keradus** [comment]
yes, with 2.12.10 milestone.

**Comment by brendt** [comment]
Could you explain how it should be configured in order to work? I'm using `v2.15.1`, and this is my current configuration.

```php
        'method_argument_space' => [
            'on_multiline' => 'ensure_fully_multiline',
            'keep_multiple_spaces_after_comma' => true,
        ]
```

Running CSFixer still results in wrongly aligned code:

```php
        $updateNumberAction->execute(
            Number::find($validated['owner_number']['id']),
            $validated['owner_number'],
            );
```

**Comment by keradus** [comment]
this PR was not touching anything related to your example.
for your example, it doesn't matter do you have an comma after last parameter or not, `)` will not get realigned.

please report an dedicated issue (following all the details from issue template)

**Comment by brendt** [comment]
I'm confused, this PR addressed https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4414 and that's exactly what that issue was about?

**Comment by keradus** [comment]
i took a deeper look

4414 is `MethodArgumentSpaceFixer - Multiline trailing comma in function calls`.
and this PR actually added support for handling comma after last parameter indeed.

then, bug with non-fixed `)` placing if it's on next line exists regardless one has that comma or not.
claiming that `)` misplaced was due to comma creates the confusion and only comma-support was added

**Comment by keradus** [comment]
ref #4436

**Comment by SpacePossum** [comment]
you case is in the unit tests:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/4420/files#diff-c2d4cc2096dd614d141880c5036ee9b0R965

but maybe with different configuration, is that the issue here?

**Comment by keradus** [comment]
it's different - in your example closing `)` was placed properly and not required to be fixed.
look at example in my pr - #4436

**Pull request merged**